### ISSUE TITLE
Unable to use the update channel selector with the new version of NVDA

### ISSUE THREAD

**Issue by GeorgeWu3** [nvda-es/updateChannel#23]
Hi
When I change the channel to alpha,beta,or Stable, it doesn't search for the update info, it's always keep on saying already updated no matter what channel I am at, so I wonder if you can fix this please? It used to work fine for me before that API breaking changes was occur in NVDA 2025.1.
thanks, And that issue frustrates and annoys me for the past couple months and hopefully it will be solved soon.
For example now I'm using the NVDA 2025.1.2 RC and I'm trying to return to stable but when I go to check for updates, it says no updates available.
